index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class="navbar">
```

Comment:
- **Try to follow the [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)**

- [ ] **Enclose the navbar inside a `<header>` tag**